test/generators/typescript/TypeScriptGenerator.spec.ts

```diff
@@ -14,16 +14,16 @@ describe('TypeScriptGenerator', () => {
       }
     };
     const expected = `export class Address {
-  private _state?: "Texas" | "Alabama" | "California" | "other";
+  private _state?: State;
```

Comment:
You should also test here the shape of `State` enum :) As I remember with

```ts
    const model = inputModel.models['State'];
```

function.

Reply:
Technically, this test should be removed as we no longer have `union property type` with enums 🤔

Reply:
I have removed it.